File: Sorting/selection_sort/algo-1/1.py
# Implementing Selection Sort in Pyhton using Basic Synatx
# Input the Size Of Array
N = int(input())
# Declare a empty List to store elements
A = []
# To read the elements
for i in range(N):
     # Adding element in list using list.append()
     A.append(int(input()))

# Selection Sort Algorithm
# we can also use numpy.arange()
for i in range(N-1):
        # Let i-th element be minimum
        key = i;	
        j = i+1
        while j<N: 
            if A[j] < A[key]:
                        key = j
            j = j+1
                
        # swap(A[i],A[key])
        temp = A[i];
        A[i] = A[key];
        A[key] = temp;	

# Output
for i in range(N):
    print(A[i],end=" ")

File: Sorting/selection_sort/algo-1/1.php
<?php
	// Selection Sort Using php 
	// Read Input from standard input file 
	$handle = fopen ("php://stdin","r");
	// Read Size Of Array
	$N = (int)fgets($handle);
	$A = array();
	// Input Array elements
	for($i=0;$i<$N;$i++){
		// We can also use array_push($A,$number)
		$A[$i] = (int)fgets($handle);	
	}
	// Selection Sort Algorithm
	for($i=0;$i<$N-1;$i++){
		// Let i-th element be minimum
		$key = $i;	
		for($j=$i+1;$j<$N;$j++){
			if($A[$j]<$A[$key]){
				$key = $j;
			}
		}
		// swap(A[i],A[key])
		$temp = $A[$i];
		$A[$i] = $A[$key];
		$A[$key] = $temp;	
	}
	// Output
	for($i=0;$i<$N;$i++){
		echo $A[$i]." ";
	}
?>
